package.json

```diff
@@ -32,14 +32,14 @@
     "babel-loader": "6.2.4",
     "babel-preset-es2015-native-modules": "6.9.0",
     "babel-preset-react": "6.11.1",
-    "electron-builder": "5.14.2",
-    "electron-prebuilt": "1.2.8",
+    "electron-builder": "5.16.0",
+    "electron-prebuilt": "1.3.0",
```

Comment:
@freebroccolo that'd be amazing but it's difficult because `../package` doesn't get shipped. `electron-builder` only ships `app/`. We'd have to copy `package.json` into `package-dev.json` and place it inside `app/` at build time or something like that. Doesn't sound great. Maybe there's another way of obtaining that version number in runtime